syft/distro/type.go

```diff
@@ -1,64 +0,0 @@
-package distro
-
-// Type represents the different Linux distribution options
-type Type string
-
-const (
```

Comment:
I think these constants might still be useful, even if they aren't of type `Type` any more — it's nice to govern the expected string values by referencing a centrally defined value, rather than relying on the value (e.g. `"opensuse-leap"` to be specified correctly at every use. What are your thoughts?

Reply:
Constants have now been kept in favor of a more gradual change to move the functionality to Grype.